src/java/test/org/apache/zookeeper/test/FourLetterWordsTest.java

```diff
@@ -167,6 +167,8 @@ public void testValidateStatOutput() throws Exception {
         line = in.readLine();
         Assert.assertTrue(Pattern.matches("^Mode: .*$", line));
         line = in.readLine();
+        Assert.assertTrue(Pattern.matches("^Fsync threshold exceeded: \\d+$", line));
```

Comment:
any chance we can add this to the very end of the output? We weren't very clear on order and in some cases existing monitoring may be expecting the original order. Appending helps wrt b/w compat.

Reply:
Well, it can be achieved, but needs some extra functions in ServerStats:
-toStringWithoutFsync() 
-fsyncToString()
(names are not final :) )

Because StatCommand first calls serverStats.toString(), and in serverStats, new Fsync command is already at the end. But in the commandRun() in StatCommand, the Node count is appended after the whole serverStats.toString() call.

Is this OK with you @phunt ?

Reply:
That's pretty ugly, esp. if there are changes in the future.

Given the Monitor command is available since 3.4.0 and handles (documented) such changes correctly perhaps we should just add this to monitor and not add it to stat. If a user wants the additional information they can use mntr instead.

What do you think @nkalmar ?

The other benefit is that it will integrate into 3.4 more easily - i.e. fully b/w compat w/o any worries.

Reply:
Yes, it would have been an ugly solution, that's why I didn't go ahead with it.
Okay, sounds good to me, I will make the changes.

Thanks!